### PR TITLE
DEVX-2699: Clarify cp-demo Gitpod support model is best effort

### PR DESCRIPTION
**Pull request by ybyzek**
### Description 

https://confluentinc.atlassian.net/browse/DEVX-2699

_What behavior does this PR change, and why?_

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

- [x] Documentation

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

**Comment by MosheBlumbergX** [comment]
lgtm

File: docs/on-prem.rst [hunk 1]
```diff
@@ -61,6 +65,7 @@ If you are using Docker:
 Gitpod
 ~~~~~~
 
+This demo is enabled to run with Gitpod, but support for the Gitpod workflow is best effort from the `community <https://github.com/confluentinc/cp-demo>`__.
```
Comment (rspurgeon) [inline docs/on-prem.rst:68]: I'm not sure, but would it be best to point users directly to the GH Issues page?
https://github.com/confluentinc/cp-demo/issues

Comment (ybyzek) [reply docs/on-prem.rst:68]: Ack, changed to issues

**Review by vdesabou — APPROVED** [review]
LGTM

**Review by rspurgeon — APPROVED** [review]
LGTM

**Pull request merged**